packages/oc-template-react-compiler/lib/viewTemplate.js

```diff
@@ -8,7 +8,9 @@ const viewTemplate = ({
   var modelHTML =  model.__html ? model.__html : '';
   var staticPath = model.reactComponent.props._staticPath;
   var props = JSON.stringify(model.reactComponent.props);
-  return '<div id="${reactRoot}" class="${reactRoot}">' + modelHTML + '</div>' +
+  var randomId = Math.random() * 10000000;
+  var reactRoot = "${reactRoot}" + '-' + randomId;
```

Comment:
If we just override the reactRoot variable, we change both the `id` and `class` attribute in the markup. I don't think we need to change the class too as that could break some stuff. I propose we change only the id: 

```suggestion
  var reactRootId = "${reactRoot}-" + randomId;
```